_rules/audio-transcript-2eb176.md

```diff
@@ -156,21 +165,6 @@ This `audio` element has native player controls and a [non-visible][visible] int
 </html>
 ```
 
-#### Failed Example 6
```

Comment:
I'd like to have a Passed example with a transcript that is only visible but not included in the accessibility tree.
This serves as documentation that the CG and TF have reached the conclusion that it is not necessarily a failure of 1.2.1.

If inaccessible transcript is always a failure of 1.2.1, I don't think we should remove it for the rule. We can add 1.3.1 as a secondary requirement if needed (to allow implementations that report it to have a consistent implementation), or we need a separate rule checking that transcripts are accessible (but I don't see plan for that other rule yet 🤔).

Reply:
The assumption states the transcript is part of the accessibility tree so is this still true?

Reply:
Agree to keep this example as passed

Reply:
Moved up :)